odbc_interface.c

```diff
@@ -1389,6 +1389,12 @@ SQLNativeSql (SQLHDBC ConnectionHandle,
 
   odbc_free_diag (((ODBC_CONNECTION *) ConnectionHandle)->diag, RESET);
 
+  if (TextLength2Ptr == NULL)
+    {
+      odbc_set_diag (((ODBC_CONNECTION *) ConnectionHandle)->diag, "HY009", 0, "NULL pointer: TextLength2Ptr");
```

Comment:
How about ignoring if TextLength2Ptr is NULL?

Reply:
It would be better to just ignore it and return it. However, I want to keep the selection this time for the following reasons:
1. A build of this fix has been sent to a customer and has already been verified, and if we make a change, we need to run this cycle again.
2. There is no clear definition of this part in the definition of ODBC API. Your opinion may be more reasonable.
3. A test case has already been written based on the error return.
4. The choice of response to this error seems trivial.

Anyway, thanks for your comment.